tests/Integration/Infrastructure/Controller/Regulation/RegulationDetailControllerTest.php

```diff
@@ -36,7 +36,7 @@ public function testDraftRegulationDetail(): void
         $this->assertSame('Route du Grand Brossais', $location->filter('h3')->text());
         $this->assertSame('Savenay (44260)', $location->filter('li')->eq(0)->text());
         $this->assertSame('Route du Grand Brossais du n° 15 au n° 37bis', $location->filter('li')->eq(1)->text());
-        $this->assertSame('Circulation interdite du lundi au mardi de 08h00 à 22h00 pour tous les véhicules', $location->filter('li')->eq(2)->text());
+        $this->assertSame('Vitesse Limitée à 50 Km/h du lundi au mardi de 08h00 à 22h00 pour tous les véhicules', $location->filter('li')->eq(2)->text());
```

Comment:
Au niveau des majuscules, Aurélie était d'accord pour cet affichage :

```suggestion
        $this->assertSame('Vitesse limitée à 50 km/h du lundi au mardi de 08h00 à 22h00 pour tous les véhicules', $location->filter('li')->eq(2)->text());
```

https://www.figma.com/file/YuT6Uh4wxe90U8LaPLlDzO?node-id=2372:137944&mode=design#547587111